packages/components/src/components/input-field/input-field.jsx

```diff
@@ -132,7 +132,7 @@ const InputField = ({
         if (max_is_disabled) return;
         let increment_value;
 
-        const current_value = local_value || value;
+        const current_value = +(local_value || value);
```

Comment:
```suggestion
        const current_value = Number(local_value || value);
```
Just for better readability since not all devs know what `+` does 🙇🏻